SPECS/python-pytest-cov/python-pytest-cov.signatures.json

```diff
@@ -0,0 +1,5 @@
+{
+    "Signatures": {
```

Comment:
>   [](http://example.com/codeflow?start=0&length=2)

Nit: Can we use 1 space per indentation, for these signature files? Seems to be standard for these.